engines/native/sandbox.go

```diff
@@ -78,26 +90,22 @@ func newSandbox(b *sandboxBuilder) (*sandbox, error) {
 	process, err := system.StartProcess(system.ProcessOptions{
 		Arguments:     b.payload.Command,
 		Environment:   env,
-		WorkingFolder: workingFolder.Path(),
+		WorkingFolder: user.Home(),
 		Owner:         user,
 		Stdout:        ioext.WriteNopCloser(b.context.LogDrain()),
 		// Stderr defaults to Stdout when not specified
 	})
 	if err != nil {
-		if b.engine.config.CreateUser {
-			user.Remove()
-		}
-
-		workingFolder.Remove()
-
 		// StartProcess provides human-readable error messages (see docs)
 		// We'll convert it to a MalformedPayloadError
-		return nil, engines.NewMalformedPayloadError(
+		err = engines.NewMalformedPayloadError(
 			"Unable to start specified command: ", b.payload.Command, "error: ", err,
 		)
+		b.context.LogError(err)
```

Comment:
Note. things like this is not useful...
If we return malformed payload, the worker package is responsible for logging that error...
That said it would make a lot of sense to use `b.context.LogError(...` to log things that aren't part of the malformed payload error... such as a text explaining what happened... But I suspect that's not something we do initially.